coffee/lightbox-plans.controller.coffee

```diff
@@ -90,7 +90,8 @@ class LightboxPlansController
             })
 
     _onSuccessSelectPlan: () ->
-        @confirm.notify('success', 'OK, te has suscrito al plan correctamente', '', 5000)
+        message = @translate.instant("SUBSCRIPTIONS.SELECT_PLAN.SUCCESS")
```

Comment:
Cannot read property 'instant' of undefined
The service isn't injected so it can't work...